snippets/validations.cson

```diff
@@ -37,3 +37,7 @@
   'validates_uniqueness_of_scope':
     'prefix': 'vuos'
     'body': 'validates_uniqueness_of :${1:field}, scope: :${2:other_field}'
+
+  'validates_presence_of':
```

Comment:
Can you change this to `validates_confirmation_of`? just for better usage with autocomplete :sweat_smile: